src/modality-specific-files/task-events.md

```diff
@@ -98,12 +99,42 @@ In the accompanying JSON sidecar, the `trial_type` column might look as follows:
             "start": "A red square is displayed to indicate starting",
             "stop": "A blue square is displayed to indicate stopping"
         }
+    },
+    "channel": {
+        "Description": "Channel(s) associated with the event",
+        "Delimiter": "|"
+    },
+    "artifact": {
+        "Description": "Description of an artifact."
     }
 }
 ```
 
-Note that all other columns SHOULD also be described but are omitted for the
-sake of brevity.
+Note that in the example above:
+
+1.  Only a subset of columns are described for the sake of brevity.
+    In a real dataset, all other columns SHOULD also be described.
+
+1.  The `channel` column contains a list of values that are separated
+    by a non-default delimiter (that is, *not* with a comma `,`).
+    See the `Delimiter` field in the accompanying JSON sidecar.
+    Thus, the channels related to the event in the third row of the example
+    are called `F,1`, `F,2`, and `Cz`.
+
+1.  The example contains a column called `artifact`.
```

Comment:
I think that the two examples that relate to this should have this column either called "artifact" or "annot".